generated/google_cloud_ai_generativelanguage_v1beta/test/generate_test.dart

```diff
@@ -0,0 +1,70 @@
+// Copyright 2025 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@TestOn('vm')
+library generative_test;
+
+import 'package:test/test.dart';
+
+import 'package:google_cloud_ai_generativelanguage_v1beta/generativelanguage.dart';
+import 'package:googleapis_auth/auth_io.dart' as auth;
+import 'package:test_utils/test_http_client.dart';
+
+void main() async {
+  late GenerativeService generativeService;
+  late TestHttpClient testClient;
+
+  group('generative', () {
+    setUp(() async {
+      final authClient = () async =>
+          await auth.clientViaApplicationDefaultCredentials(
+            scopes: [
+              'https://www.googleapis.com/auth/cloud-platform',
+              'https://www.googleapis.com/auth/generative-language.retriever',
+            ],
+          );
+
+      testClient = await TestHttpClient.fromEnvironment(authClient);
+      generativeService = GenerativeService(client: testClient);
+    });
+
+    tearDown(() => generativeService.close());
+    test('streamed', () async {
+      await testClient.startTest(
+        'google_cloud_ai_generativelanguage_v1beta',
+        'generative_streamed',
+      );
+
+      final request = GenerateContentRequest(
+        model: 'models/gemini-2.5-flash',
+        contents: [
+          Content(
+            parts: [Part(text: "Explain how AI works in extensive detail")],
```

Comment:
[nit] Other strings in this library use single quotes, aim for consistency. Consider enabling the `prefer_single_quotes` lint to use dart fix to keep things consistent automatically.